string action_name
string[] options
string[] predicates

---
bool result
string[] parameters
string[] precondition
string[] effect
string[] primitives

string[] controller
string[] group
string[] planner
